src/daemon.py

```diff
@@ -30,100 +29,74 @@ def compile_yara(job_hash: str) -> Any:
     return rule
 
 
-def get_list_name(priority: str) -> str:
-    if priority == "low":
-        return "list-yara-low"
-    elif priority == "medium":
-        return "list-yara-medium"
-    else:
-        return "list-yara-high"
-
-
 def collect_expired_jobs() -> None:
     if config.JOB_EXPIRATION_MINUTES <= 0:
         return
 
     exp_time = int(60 * config.JOB_EXPIRATION_MINUTES)  # conversion to seconds
-    job_hashes = []
+    job_ids: List[JobId] = []
 
-    for job_hash in redis.keys("job:*"):
-        job_hashes.append(job_hash[4:])
+    for job in redis.get_job_ids():
```

Comment:
Is `job_ids` used anywhere else besides the loop below? Seems like it isn't really needed

Reply:
Do you mean, why is it a temporary varluable variable instead of `for job in redis.get_job_ids()`? I assumed it's to avoid race conditions of some kind, but on a second thought it doesn't make sense. I tried to avoid doing any logical changes, but in this case it may make sense :thinking: .